packages/core/src/types.ts

```diff
@@ -407,6 +407,11 @@ export interface ConfigBase<Theme extends {} = {}> {
    * @default false
    */
   details?: boolean
+
+  /**
+   * Excluded pseudo elements when moving to the end of selector
+   */
+  unsortedPseudoElements?: string[]
```

Comment:
In my opinion, we don't need to provide this configuration to end users. I can't think of any scenario where this can be helpful 🤔

Reply:
The function location in `generator.applyVariants` is tricky to add some kind of constants without config, except maybe importing it directly into `movePseudoElementsEnd` ?

```ts
import { DEFAULT_UNSORTED_PSEUDO_ELEMENTS } from './constants'

export function movePseudoElementsEnd(selector: string) {
  const excludedPseudo = DEFAULT_UNSORTED_PSEUDO_ELEMENTS

```

Reply:
We can inline it right before the function definition for better readability.

Reply:
see #2190